src/instructions/proctored_exam/index.js

```diff
@@ -4,5 +4,6 @@ export { default as SubmitProctoredExamInstructions } from './SubmitProctoredExa
 export { default as SubmittedProctoredExamInstructions } from './SubmittedProctoredExamInstructions';
 export { default as VerificationProctoredExamInstructions } from './VerificationProctoredExamInstructions';
 export { default as VerifiedProctoredExamInstructions } from './VerifiedProctoredExamInstructions';
-export { default as DownloadSoftwareProctoredExamInstructions } from './download-instructions';
+export { default as DownloadSoftwareProctoredExamInstructions } from './download-instructions/index';
```

Comment:
Is it necessary to have the `index` here? `index.js` should be as a default file in the path.

Reply:
Yes, you're right, that's definitely not necessary, updated imports in new commit